core/internal/api/api.go

```diff
@@ -14,6 +19,12 @@ import (
 type Backend struct {
```

Comment:
i'm not sure about the Backend name, but all that comes to mind is API so it's not really better...

Reply:
maybe EndPoint? don't know also weird..

Reply:
(can ignore this...)